server/src/main/scala/actors/persistent/staffing/StaffingUtil.scala

```diff
@@ -227,4 +228,46 @@ object StaffingUtil {
 
   }
 
+  def addAssignments(newShift: Shift,
```

Comment:
This needs a better name. I'm not sure what it does but it doesn't add assignments. Maybe it merges assignments with overlapping shifts?